morecantile/models.py

```diff
@@ -162,6 +168,11 @@ def _invert_axis(self) -> bool:
         """Check if CRS has inverted AXIS (lat,lon) instead of (lon,lat)."""
         return crs_axis_inverted(self.crs)
 
+    @property
+    def quadkey_support(self) -> bool:
+        """Indicator if the Tile Matrix Set supports the creation of quadkeys"""
+        return check_quadkey_support(self.tileMatrix)
```

Comment:
This seems expensive to run every time we call `quadkey` or `quadkey_to_tile`.  Given tile matrix sets should be static (we won't be adding new tile matrices after a `TileMatrixSet` is isntantiated), I think we can get away with either making this an instance attribute or caching the output of this property the first time it's accessed.

This way we don't have to iterate through all the tile matrixes every time we want to do something with a quadkey.

Reply:
You are right! I made it an instance attribute.